frontend/src/views/account-mgmt/ManageOrganizationView.vue

```diff
@@ -147,8 +170,58 @@ export default {
      * Open the Change Request dialog
      */
     openChangeRequestDialog() {
-      this.setWarningAlert('This feature is not yet implemented')
+      this.setWarningAlert('This feature will be implemented in a future sprint')
+    },
+
+    async saveOrganization(updatedOrganization) {
+      try {
+        await OrganizationService.updateOrganization(this.organization.organizationId, updatedOrganization)
+        this.organization = updatedOrganization
+      } catch (error) {
+        this.setFailureAlert('Failed update Inclusion Policy on Organization: ' + this.organization.organizationId, error)
+        return
+      }
     },
+
+    async saveInclusionPolicyData(updatedOrganization, documentsToUpload, documentsToDelete) {
+      this.loadingInclusionPolicy = true
+      this.documentsToUpload = documentsToUpload
+      this.documentsToDelete = documentsToDelete
+      try {
+        await this.saveOrganization(updatedOrganization)
+        await this.processDocuments()
+        await this.getInclusionPolicyDocuments()
+        this.$refs.organizationInfo.resetData()
```

Comment:
This can be done in the OrganizationInfo.vue save.

Reply:
Fixed.